indicator_creation_rules/new_user_account_logging.py

```diff
@@ -13,7 +13,7 @@ def rule(event):
 
     user_event_id = f"new_user_{event.get('p_row_id')}"
     new_user = event.udm("user")
-    new_account = event.udm("user_account_id")
+    new_account = event.udm("user_account_id") or "Unknown Account"
```

Comment:
nit: standardize missing fields

```suggestion
    new_account = event.udm("user_account_id") or "<UNKNOWN_ACCOUNT>"
```